server/common/thrift/src/main/thrift-java/org/kaaproject/kaa/server/common/thrift/gen/operations/EndpointStateUpdate.java

```diff
@@ -556,7 +556,7 @@ public String toString() {
     if (this.endpointKey == null) {
       sb.append("null");
     } else {
-      sb.append(this.endpointKey);
+      sb.append(Hex.encodeHexString(this.endpointKey.array()));
```

Comment:
This is autogenerated class your changes will be overwritten after next generation.